new/PRS13411GondareH.xml

```diff
@@ -45,17 +45,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">Gondare Habtu</persName>
+                    <persName xml:lang="gez" xml:id="n1">ጎንደሬ፡ ሀብቱ፡</persName>
```

Comment:
maybe Gondare should be better be marked up as addName...

Reply:
Like that?